cli/conf.py

```diff
@@ -105,6 +123,81 @@ def parse_args(cls, module_name, config, args=None):
 
         return res_args
 
+
+class SpecLoader(object):
+    """
+    Responsible to load spec files.
+    """
+    SPEC_EXTENSION = '.spec'
+    LOOKUP_DICT = {
+        '__DEFAULT__': "default"
+    }
+
+    TRIM_PARAMS = ['default', 'required']
+
+    @classmethod
+    def load(cls, module_name, config):
+        """
+        Loads the spec file as dict.
+        """
+        clg_options = cls._get_specs(module_name, config)
+        global_options, subcommands_options = cls._get_options(clg_options)
+        return clg_options, global_options, subcommands_options
+
+    @classmethod
+    def _get_options(cls, spec_dict, lookup=False, trim=False):
```

Comment:
This need some documentation.
What's the diff between `_get_options` and `__get_options`?